HintList.py

```diff
@@ -520,51 +520,51 @@ def tokens_required_by_settings(world):
-    'LH Lake Lab Pool':                                            ("inside a #lakeside lab# a person and a spider hold respectively...^", None, 'dual'),
-    'LH Adult Bean Destination Checks':                            ("#riding the bean platform# of the lake to the lab's roof and the fishing hole leads to...^", None, 'dual'),
-    'GV Pieces of Heart Ledges':                                   ("within the #valley#, the crate and waterfall conceal respectively...^", None, 'dual'),
-    'GF Horseback Archery Rewards':                                ("at the #Gerudo Horseback Archery#, scoring 1000 and 1500 gives respectively...^", None, 'dual'),
-    'Colossus Nighttime GS':                                       ("#outside the Desert Colossus# spiders on palm trees and hills hold respectively...^", None, 'dual'),
-    'Graveyard Dampe Race Rewards':                                ("racing #Dampe's ghost# yields...^", None, 'dual'),
-    'Graveyard Royal Family Tomb Contents':                        ("inside the #Royal Family Tomb#, darkness and Redeads guard respectively...^", None, 'dual'),
-    'DMC Child Upper Checks':                                      ("at #the crater# a spider in a crate and a single scrub hold respectively...^", None, 'dual'),
-    'Haunted Wasteland Checks':                                    ("#deep in the wasteland# a chest and a spider hold respectively...^", None, 'dual'),
+    'Deku Theater Rewards':                                        ("the #Skull Mask and Mask of Truth# reward...^", None, 'dual'),
+    'HF Ocarina of Time Retrieval':                                ("the #Ocarina of Time# rewards both...^", None, 'dual'),
+    'HF Valley Grotto':                                            ("in a grotto with a #spider and a cow# you will find...^", None, 'dual'),
+    'Market Bombchu Bowling Rewards':                              ("at the #Bombchu Bowling Alley#, you will be rewarded with...^", None, 'dual'),
+    'ZR Frogs Rewards':                                            ("the #Frogs of Zora River#, will reward you with...^", None, 'dual'),
+    'LH Lake Lab Pool':                                            ("inside the #lakeside lab# a person and a spider hold...^", None, 'dual'),
+    'LH Adult Bean Destination Checks':                            ("#riding the bean in Lake Hylia# leads to...^", None, 'dual'),
+    'GV Pieces of Heart Ledges':                                   ("within the #valley#, the crate and waterfall conceal...^", None, 'dual'),
+    'GF Horseback Archery Rewards':                                ("the #Gerudo Horseback Archery#, rewards...^", None, 'dual'),
+    'Colossus Nighttime GS':                                       ("#at the Desert Colossus#, skulltulas at night hold...^", None, 'dual'),
+    'Graveyard Dampe Race Rewards':                                ("racing #Dampe's ghost# rewards...^", None, 'dual'),
```

Comment:
Might as well fix this too…
```suggestion
    'Graveyard Dampe Race Rewards':                                ("racing #Dampé's ghost# rewards...^", None, 'dual'),
```